src/EventStore.Core.Tests/Services/Storage/ReadIndexTestScenario.cs

```diff
@@ -109,8 +111,8 @@ public override void TestFixtureTearDown()
 
         protected abstract void WriteTestScenario();
 
-        protected EventRecord WriteSingleEvent(string eventStreamId, 
-                                               int eventNumber, 
+        protected EventRecord WriteSingleEvent(string eventStreamId,
```

Comment:
Whitespace changes = correct, but probably should be on a different PR to avoid the noise here!